frontend/src/AddSeries/ImportSeries/Import/ImportSeriesTable.tsx

```diff
@@ -78,7 +80,10 @@ function ImportSeriesTable({
   const listRef = useRef<FixedSizeList<RowItemData>>(null);
   const initialUnmappedFolders = useRef(unmappedFolders);
   const previousItems = usePrevious(items);
-  const { allSelected, allUnselected, selectedState } = selectState;
+  const { allUnselected, selectedState } = selectState;
+  const allSelected = useSelector((state: AppState) =>
```

Comment:
Instead of a separate selector this should be made more generic so the logic can be reused in other places we use this state. Probably something similar to how `toggleSelected` works, then uncheck it and add it to record / state that has the ID and `true` so we can look through all items and confirm it's either checked or disabled.

Reply:
`allSelected` would still come from `useSelect`, `useSelect` would also include disabled items and `allSelected` would derive it's value from items being selected as well as items that are disabled. Similar to selecting an item items that are disabled would dispatch that to the state in the context.